big-data-cluster extension fails to load on Mac Insiders build

Today's packaged build has this issue, seems due to webpack issue

```
[Extension Host] Activating extension `Microsoft.big-data-cluster` failed:  Cannot find module '/Applications/Azure Data Studio - Insiders.app/Contents/Resources/app/extensions/big-data-cluster/dist/extension' (at define._activateExtension.newlyActivatingExtension._host.actualActivateExtension.then (/Applications/Azure Data Studio - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:34786:25))
console.(anonymous function) @ /Applications/Azure Data Studio - Insiders.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:2280
```

--- a/extensions/big-data-cluster/extension.webpack.config.js
+++ b/extensions/big-data-cluster/extension.webpack.config.js
@@ -12,7 +12,7 @@ const withDefaults = require('../shared.webpack.config');
 module.exports = withDefaults({
 	context: __dirname,
 	entry: {
-		main: './src/extension.ts'
+		extension: './src/extension.ts'
 	},
 	externals: {
 	}

